YoYoProject/Controllers/GMSound.cs

```diff
@@ -139,11 +139,31 @@ internal override ModelBase Serialize()
 
         internal override void Deserialize(ModelBase model)
         {
-            // TODO Implement
             var soundModel = (GMSoundModel)model;
 
             Id = soundModel.id;
             Name = soundModel.name;
+            CompressionKind = soundModel.kind;
+            Volume = soundModel.volume;
+            PreLoad = soundModel.preload;
+            BitRate = soundModel.bitRate;
+            SampleRate = soundModel.sampleRate;
+            Type = soundModel.type;
+            BitDepth = soundModel.bitDepth;
+
+        }
+
+        internal override void FinalizeDeserialization(ModelBase model)
+        {
+            var soundModel = (GMSoundModel)model;
+            Guid audiogrpGuid = soundModel.audioGroundGuid;
```

Comment:
Nothing wrong with using explicit types generally. However, the codebase already standardized using 'var' when type of the right-hand side is obvious. In this case "Guid" being suffixed makes the type obvious.

Change to var.